nservicebus/bridge/configuration.md

```diff
@@ -55,17 +55,46 @@ The queues can be created using one of the following methods:
 
 - Provisioning them manually using the tooling provided by the queuing system.
 - Using the queue creation tooling provided by Particular Software if one exists for the transports being used. See the [individual transports documentation](/transports/) for more details.
-- Configuring the bridge to create queues automatically as described in the next section.
+- Configuring the bridge to create queues of proxied endpoints automatically as described in the next section.
```

Comment:
It might be worth explaining what is meant by proxied endpoints at this point as this is the first mention of it in the sample. Alternatively, if we use the term in the bridge docs, we should link to that definition from here.

Reply:
We use the term in the bridge documentation here https://docs.particular.net/nservicebus/bridge/configuration#registering-endpoints